webapp/www/templates/home.html

```diff
@@ -45,43 +45,43 @@
 <fullpage class="instant-size" page-class="page">
```

Comment:
Maybe just convert this from fullpage to list and show Juliana/Yawen/Adam for design feedback and then decide whether we want a different fullpage library as a next step?